src/components/DownloadBulkNibrs.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+
+const bulkNibrs = 'http://s3-us-gov-west-1.amazonaws.com/cg-d3f0433b-a53e-4934-8b94-c678aa2cbaf3'
+const createBulkNibrsUrl = (year, state) => (
+  `${bulkNibrs}/${year}/${state}-${year}.zip`
+)
+
+const downloadBulkNibrs = (year, state) => {
+  const a = document.createElement('a')
+  a.href = createBulkNibrsUrl(year, state)
+  a.click()
+}
+
+class DownloadBulkNibrs extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      place: 48,
+      year: null,
+    }
+
+    this.handleClick = ::this.handleClick
+    this.handleSelectChange = ::this.handleSelectChange
+  }
+
+  handleClick(e) {
+    e.preventDefault()
+    const { place, year } = this.state
+    downloadBulkNibrs(year, place)
+  }
+
+  handleSelectChange(e) {
+    switch (e.target.id) {
+      case 'nibrs-state':
+        this.setState({ place: e.target.value })
+        break
+      case 'nibrs-year':
+        this.setState({ year: e.target.value })
+        break
+      default:
+        break
+    }
+  }
+
+  render() {
+    const border = 'border-bottom border-blue-lighter'
+    return (
+      <div>
+        <h2 className={`${border} mb2 fs-22 sm-fs-32`}>
+          Download incident data by state and year
+        </h2>
+        <p className='fs-24'>
+          To view all attributes of incident data (NIBRS) join CSV downloads into a pivot table.
+        </p>
+        <form className='p2 sm-p4 bg-blue-lighter'>
+          <fielset className='clearfix'>
+            <legend>Choose a file to download</legend>
+            <div className='col col-5'>
+              <label htmlFor='nibrs-state'>Select a place</label>
+              <select
+                className='sm-fs-18 bold field'
+                id='nibrs-state'
+                onChange={this.handleSelectChange}
+              >
+                <option value='01'>AL</option>
```

Comment:
that is correct @cacraig, we are not providing bulk NIBRs data at the national level; only for individual states that report NIBRS. 

@jeremiak to your point, we should only provide reports for the states that submit NIBRS data. So the drop-down/UI should be limited to those states. We may also want to provide a clarifying statement on the page about why data in this format is only available for certain states and not others. 

cc: @nicoleslaw